src/Focusable.ts

```diff
@@ -6,13 +6,14 @@
 import { getTabsterOnElement } from "./Instance";
 import { RootAPI } from "./Root";
 import * as Types from "./Types";
+import { FocusableSelector } from "./Consts";
```

Comment:
how about we adopt ALL_CAPS_KEBAB  as a convention for constants? that avoids a naming clash with the corresponding type and makes it clear in the code that constants are being used

Reply:
I've ALL_CAPS_KEBABed them.